### PR TITLE
Update History of osu! 2014

### PR DESCRIPTION
**Pull request by MilkyIQ**
Cleans up and updates the original article with more information on different months throughout 2014. I only used information detailed in that year's newsposts, so some information regarding gameplay updates may have been left out. I made a point to ignore any "Monthly ____ Contests" aside from firsts for obvious reasons.

Part of efforts detailed in #4871; **follows citation conventions detailed in #4911**

*I think it might be better to reorganize the articles to detail events chronologically, but separated in sections by topic rather than by month, but i kept it as is just to make cleanup and updates and easier for the time being*

### CODE REVIEW

**Review by peppy — DISMISSED** [review]
This reads much better. I think the information looks correct and have no qualms related to it. I haven't done a thorough grammar / style pass so won't merge.

File: wiki/History_of_osu!/2014/en.md [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+October brought a handful of new additions to the base game, including a new sign in screen and new system for scrolling in [osu!mania](/wiki/Game_mode/osu!mania) based off of beatmap BPM, new intro and outro sequences, and a custom theme song that was to be played on startup (as opposed to a random beatmap). <!--citation needed-->
 
-Lastly, leading to peppy making a tough decision to release *osu!test.exe* (now *osu!cuttingedge*) for **anyone**. This was to allow non-osu!supporters to *debug* osu! where the newest (or latest) builds of osu! would be updated to any client. That was, if they're wanting to use it's unstable/experimental features. Every function was given in osu!cuttingedge **except** for multiplayer (osu!supporter was required).
+Additionally, [peppy](https://osu.ppy.sh/users/2) also released the "osu!test" build (now known as "Cutting Edge) to all players, as opposed to just [osu!supporters](https://osu.ppy.sh/home/support) in an attempt to help debugging efforts. <!--citation needed-->
```
Comment (Joehuu) [inline wiki/History_of_osu!/2014/en.md:31]:
```suggestion
Additionally, [peppy](https://osu.ppy.sh/users/2) also released the "osu!test" build (now known as "Cutting Edge") to all players, as opposed to just [osu!supporters](https://osu.ppy.sh/home/support) in an attempt to help debugging efforts. <!--citation needed-->
```

File: wiki/History_of_osu!/2014/en.md [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
 
-## December
+## References
 
-"Merry Christmas" as peppy wrote with an update that had snow (current game mode icon) falling in the main screen!
+1. [osu! newspost [peppy] (20 January 2014) - "Performance Ranking Maintenance"](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)
+2. [osu! newspost [Tom94] (20 January 2014) - "Performance Ranking Maintenance"](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)
```
Comment (Joehuu) [inline wiki/History_of_osu!/2014/en.md:36]: is the first and second reference supposed to be similar?

**Review by Joehuu — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/2014/en.md (hunk 2 above)
Comment (MilkyIQ) [reply wiki/History_of_osu!/2014/en.md:36]: No, this was just an oversight by me. Should be this
```suggestion
2. [osu! newspost [Tom94] (26 January 2014) - "New Performance Ranking"](https://osu.ppy.sh/home/news/2014-01-26-new-performance-ranking)
```

**Review by Joehuu — APPROVED** [review]
